### PR TITLE
DOC: Improve doc summaries in series.rst

### PR DESCRIPTION
**Pull request by mroeschke**

### CODE REVIEW

File: pandas/core/arrays/sparse/array.py [hunk 1]
```diff
@@ -1265,7 +1265,7 @@ def astype(self, dtype: AstypeArg | None = None, copy: bool = True):
 
     def map(self: SparseArrayT, mapper) -> SparseArrayT:
         """
-        Map categories using input correspondence (dict, Series, or function).
+        Map categories using an input mapping or function.
```
Comment (jbrockmendel) [inline pandas/core/arrays/sparse/array.py:1268]: OT: itd be nice to share this docstring with Categorical.map.

File: pandas/core/indexes/accessors.py [hunk 2]
```diff
@@ -193,7 +193,7 @@ class DatetimeProperties(Properties):
 
     def to_pydatetime(self) -> np.ndarray:
         """
-        Return the data as an array of native Python datetime objects.
+        Return the data as an array of :class:`datetime.datetime`.
```
Comment (jbrockmendel) [inline pandas/core/indexes/accessors.py:196]: could keep the word "objects" like you did in to_pytimedelta and a "date" above (but not time or timetz)?

**Comment by jbrockmendel** [comment]
Small comments, otherwise LGTM

File: pandas/core/indexes/accessors.py (hunk 2 above)
Comment (mroeschke) [reply pandas/core/indexes/accessors.py]: Sounds good. Changed.

File: pandas/core/arrays/datetimes.py [hunk 3]
```diff
@@ -1290,16 +1292,19 @@ def time(self) -> npt.NDArray[np.object_]:
...
         return ints_to_pydatetime(self.asi8, self.tz, box="time")
 
     @property
     def date(self) -> npt.NDArray[np.object_]:
         """
-        Returns numpy array of python datetime.date objects (namely, the date
-        part of Timestamps without timezone information).
+        Returns numpy array of python :class:`datetime.date` objects.
+
+        Namely, the date part of Timestamps without timezone information.
```
Comment (jbrockmendel) [inline pandas/core/arrays/datetimes.py:1307]: i think also without the time of day part?

File: pandas/core/arrays/datetimes.py [hunk 4]
```diff
@@ -570,7 +570,7 @@ def dtype(self) -> np.dtype | DatetimeTZDtype:  # type: ignore[override]
     @property
     def tz(self) -> tzinfo | None:
         """
-        Return timezone, if any.
+        Return the timezone.
```
Comment (jbrockmendel) [inline pandas/core/arrays/datetimes.py:573]: why no "if any"?

File: pandas/core/arrays/datetimes.py [hunk 5]
```diff
@@ -1278,7 +1278,9 @@ def day_name(self, locale=None):
     @property
     def time(self) -> npt.NDArray[np.object_]:
         """
-        Returns numpy array of datetime.time. The time part of the Timestamps.
+        Returns numpy array of :class:`datetime.time`.
```
Comment (jbrockmendel) [inline pandas/core/arrays/datetimes.py:1281]: Not using the "objects" pattern?

File: pandas/core/arrays/datetimes.py (hunk 4 above)
Comment (mroeschke) [reply pandas/core/arrays/datetimes.py:573]: IMO it sounded better with `if any` (if it returns `None` that's the timezone). Can change it back if you feel strongly about it.

Comment (jbrockmendel) [reply pandas/core/arrays/datetimes.py:573]: > if you feel strongly about it.

nope.

**Comment by jreback** [comment]
@meeseeksdev backport 1.4.x

**Pull request merged**